tests/pages/system/overview/ClusterDetails.js

```diff
@@ -0,0 +1,42 @@
+describe("Cluster Details", function() {
+  beforeEach(function() {
+    cy.configureCluster({ mesos: "cluster-overview" });
+  });
+
+  it("displays Mesos Details", function() {
+    cy.visitUrl({ url: "/" });
+    cy.get(".breadcrumb__content").contains("Dashboard");
+
+    cy.clock(new Date(2018, 1, 13).getTime());
```

Comment:
Could find a nicer solution fast. Basically we need to load the UI then rewind the clock and then visit the Overview page. Otherwise cypress gets stuck on the pre-loader.

Reply:
Time travel is always fine